### PR TITLE
i18n support (part 2)

### PR DESCRIPTION
**Pull request by Nriver**
Continue ~ #264

### CODE REVIEW

**Comment by eliandoran** [comment]
@Nriver , could you please update the language name in `src/routes/api/options.ts`, in the `getSupportedLocales` method? We would need to have the localized name of Chinese in there instead of simply "Chinese".

I would do it myself but I don't know exactly whether it's "简体中文", "中文" or something else.

**Comment by Nriver** [comment]
I noticed your comment in the code and assumed that you'd eventually use keys like "LANG_NAME": "English" or "LANG_NAME": "简体中文" within the JSON file to make it self-explanatory, avoiding the need for a hard-coded list. That's why I didn't make any changes initially.

However, since you've pointed it out, I'll go ahead and update it now.

**Comment by Nriver** [comment]
Just made a small PR here #351

**Comment by hasecilu** [comment]
Minor thing on translation files: on `attachments_actions` the `rename_attachment` key is duplicated.

File: src/public/translations/en/translation.json [hunk 1]
```diff
@@ -1119,5 +1119,116 @@
...
+        "test_title": "Sync Test",
+        "test_description": "This will test the connection and handshake to the sync server. If the sync server isn't initialized, this will set it up to sync with the local document.",
+        "test_button": "Test sync",
+        "handshake_failed": "Sync server handshake failed, error: {{message}}"
+    },
+    "api_log": {
+        "close": "Close"
+    },
+    "attachment_detail_2": {
+        "will_be_deleted_in": "This attachment will be automatically deleted in {time}}",
```
Comment (hasecilu) [inline src/public/translations/en/translation.json:1227]: Missing `{` on _time_

Comment (Nriver) [reply src/public/translations/en/translation.json]: Thanks for the review, fixed.

**Comment by Nriver** [comment]
@eliandoran We need to establish a standard for JSON file indentation. The inconsistency between using 2 and 4 spaces is causing issues. Here's an example: https://github.com/TriliumNext/Notes/pull/342/conflicts.

**Comment by Nriver** [comment]
I update the config for WebStorm json code style [.idea/codeStyles/Project.xml](https://github.com/TriliumNext/Notes/pull/342/commits/28908851579d3212d8ceed8780cbf05060199222#diff-11745b49c51a1ecde7d4d4cffda53d76594e6711ec81bf1148a05942b0438141)

**Review by JYC333 — APPROVED** [review]

**Review by eliandoran — APPROVED** [review]
LGTM.

**Pull request merged**